src/main/resources/application.yaml

```diff
@@ -33,6 +33,6 @@ integration:
 logbook:
   exclusionfilters:
     # Exclude base64-encoded content from incoming requests.
-    json-path: "{'$.bodyInformation.body' : '[base64]', '$.attachments[*].body': '[base64]'}"
+    jsonPath: "{'$.bodyInformation.body' : '[base64]', '$.attachments[*].body': '[base64]'}"
```

Comment:
This will not work (for 4.0). See documentation in: https://confluence.sundsvall.se/pages/viewpage.action?pageId=910426116